contrib/crio-wipe/README.md

```diff
@@ -0,0 +1,27 @@
+# crio-wipe
+
+crio-wipe is a program that reads CRI-O's version file, and compares it to the output of crio --version.
+If the version is deemed old enough, crio-wipe wipes container storage on the node.
+If there is no version file present, or the version file is malformatted, crio-wipe also will wipe the storage.
+
+
+crio-wipe returns 0 if no wipe happend, 1 if a wipe did happen, or 2 if an error occured that resulted in no wipe.
+
+
+crio-wipe by default assumes:
+* the containers storage directory is `/var/lib/containers`
```

Comment:
I think in the future we may want to revisit this by recording this as well or saving a copy of crio.conf, so wipe process could parse and pick the correct storage directory to wipe.

Reply:
I thought about this, but because the config location can change on runtime, I figured we should just let the user manually override default instead of trying to parse it